.github/workflows/cd.yml

```diff
@@ -17,6 +17,25 @@ jobs:
     - name: Build SDist and wheel
       run: pipx run build
 
+    - uses: actions/upload-artifact@v2
+      with:
+        path: dist/*
+
+    - name: Check metadata
+      run: pipx run twine check dist/*
+
+
+  publish:
+    needs: [dist]
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && github.event.action == 'published'
```

Comment:
Thanks for implementing this, it is great that now have a manual way of triggering the CD workflow along with upload of the packages without systematically publishing to PyPI.